manifests/plugins.pp

```diff
@@ -0,0 +1,94 @@
+# @summary Manage Sensu plugins
+#
+# Class to manage the Sensu plugins.
+#
+# @example
+#   class { 'sensu::plugins':
+#     plugins    => ['disk-checks'],
+#     extensions => ['graphite'],
+#   }
+#
+# @example
+#   class { 'sensu::plugins':
+#     plugins    => {
+#       'disk-checks' => { 'version' => 'latest' },
+#     },
+#     extensions => {
+#       'graphite' => { 'version' => 'latest' },
+#     },
+#   }
+#
+# @param package_ensure
+#   Ensure property for sensu plugins package.
+# @param package_name
+#   Name of the Sensu plugins ruby package.
+# @param dependencies
+#   Package dependencies needed to install plugins and extensions.
+#   Default is OS dependent.
+# @param plugins
+#   Plugins to install
+# @param extensions
```

Comment:
what's a plugin vs an extension?

Reply:
I'm not sure but the command `sensu-install` makes a distinction as plugins use `--plugin` flag and extensions use `--extension` flag and gem names have different prefixes.